packages/xo-server/src/xo-mixins/store.js

```diff
@@ -1,30 +1,62 @@
 import levelup from 'level-party'
 import sublevel from 'level-sublevel'
+import { defer, fromEvent } from 'promise-toolbox'
 import { ensureDir } from 'fs-extra'
 
 import { forEach, promisify } from '../utils'
 
 // ===================================================================
 
-const _levelHas = function has(key, cb) {
-  if (cb) {
-    return this.get(key, (error, value) =>
-      error ? (error.notFound ? cb(null, false) : cb(error)) : cb(null, true)
-    )
-  }
+const _levelHas = function has(key) {
+  return this.get(key).then(
+    () => true,
+    error => {
+      if (!error.notFound) {
+        throw error
+      }
+      return false
+    }
+  )
+}
 
-  try {
-    this.get(key)
-    return true
-  } catch (error) {
-    if (!error.notFound) {
-      throw error
+async function _levelClean(keep = 2e4) {
```

Comment:
IMHO `gc` makes more sense than `clean`, I better understand what it does.

Also, it might be a good idea to add a small comment above.